exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/DrillSqlWorker.java

```diff
@@ -47,6 +47,7 @@
 public class DrillSqlWorker {
   private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(DrillSqlWorker.class);
   private static final ControlsInjector injector = ControlsInjectorFactory.getInjector(DrillSqlWorker.class);
+  private static final String NEW_LINE = System.getProperty("line.separator");
```

Comment:
May be removed after changing ```StringBuilder``` to ```String.format``` in ```wrapWithAutoLimit(parser, context, sql)``` method. See comments below for details.

Reply:
Done